### DISCUSSION TITLE
Organization's Repository Starred but not Showing

### DISCUSSION THREAD

**Discussion by Volscente** [community/community#28765] [Other Feature Feedback, Questions, & Ideas]
Hi everyone,

I've been invited to an organisation with my personal email address. I used this email address also to open my GitHub account many years before being invited.

Now I have two situations not clear to me:

1. When I "Star" a repository from this organisation, the number of "Starred" repository in my profile does not increase, but I can see them in the "Stars" list under the "Stars" tab of my profile. Do you know why?
<img width="630" alt="image" src="https://user-images.githubusercontent.com/18011646/183432724-cbcf1a2e-af2b-49a9-b6db-b452a1cd1bd0.png">
2.  Even if I starred those repositories, if I'm not login into the organisation, the contributions are not showed under my Contribution Graph. I read that only starring a repository is enough to be showed in the contributions (yes, the branch is the default one and the email address is always the same), but it seems not. In this way, when I will leave the organisation, I will also lose all my contributions.

Someone can help me explaining this?

Thank you!

**Comment by melvincwng** [comment]
@Volscente, based on what I'm aware of:

Question 1 - Starring a repository **belonging to an organization** does not increase your stars count. This is different compared to starring repositories **belonging to a user**, the act of doing so (in the latter example) will increase your stars count.

Question 2 - Starring repositories **does not increase** your contributions graph count. You can increase your contribution graph count by various ways - submitting commits, creating PRs, creating issues, joining an organization and many more. But starring repos is not one of them.

Hope this clarifies, and hope you find this useful. If this has been helpful, you can 'Mark as answer' so the rest are aware this issue has been resolved.

Thanks, and take care. Have a good day ahead 😄

**Reply by Volscente** [reply]
@melvincwng thank you very much for your answers!

Question 1 - Thanks, I was not aware of that

Question 2 - Sorry, I explained myself poorly. Let me try again!

If I do not click on "Single sign-on" into my organisation from the GitHub homepage
![Screenshot 2022-08-09 at 16 20 18](https://user-images.githubusercontent.com/18011646/183678203-b00e8bc3-cadd-427a-95e1-46571f308ebb.png)


I do not see the contributions coming from commits on default branch (9 August: 0 contributions)
![Screenshot 2022-08-09 at 16 19 46](https://user-images.githubusercontent.com/18011646/183678406-04f6b312-8571-4af7-b9a2-55d354803ce7.png)


However, if I login through the "Single sign-on" into my organisation, I can see them (9 August: 3 contributions)
![Screenshot 2022-08-09 at 16 20 59](https://user-images.githubusercontent.com/18011646/183678543-92011e8c-e9b2-426b-9e2e-17908d4fd41e.png)


These are the conditions requested from GitHub and they are all met (otherwise in the last screen we won't see those 3 contributions):
![Screenshot 2022-08-09 at 16 45 03](https://user-images.githubusercontent.com/18011646/183679103-0cf1c57a-916d-40e7-a7b6-fcb8b4dc47d6.png)


So why the login into the organisation is required? When I will leave the organisation I will lose all my contributions in this way.

Thank you again for your time!

**Reply by vimpostor** [reply]
Going to the [page](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#commits) now, reveals that Github has meanwhile removed the bullet point "You have starred the repository".

So it seems one must now fork a repository so that contributions appear in your activity graph, starring is no longer sufficient.

**Comment by micha-6** [comment]
If an organization’s repository is starred but not showing up as expected, there could be several underlying issues. First, ensure that you are logged into the correct account and that the repository is indeed public or accessible according to the repository's visibility settings. Sometimes, a simple refresh of the page or logging out and back in can resolve  Additionally, check[ visit website](https://durhamnctree.com/tree-trimming-in-durham-nc/) if there are any filters or settings applied on your dashboard that might be affecting the visibility of starred repositories. If the problem persists, it may be worth investigating if there are any known issues with the platform or reaching out to support for assistance. Ensuring your account and repository settings are correctly configured can often resolve these visibility issues.
